### DISCUSSION TITLE
Enable multiple attributes when using .attr

### DISCUSSION THREAD

**Discussion by JonKaric** [livewire/livewire#8570] [3. Feature ideas]
When using the `.attr` modifier (on `wire:loading`) it's currently impossible to toggle multiple attributes.

`wire:loading.attr="disabled"`

Would be great to be able to do a comma seperated list of attributes to toggle:

`wire:loading.attr="aria-disabled, data-loading"`

Happy to make a PR for this.

**Comment by zepfietje** [comment]
Found this discussion while searching for multiple attribute support with `wire:loading.attr`.
Expected it to work exactly as you described, using a comma-separated list of attributes.

**Comment by nicolagianelli** [comment]
There is an open PR to enable multiple attributes.
If you want you can leave a comment in the PR to let Caleb and Josh know your interest in this feature.
PR: #9198
